TUM-Dev/NavigaTUM
# [MVV-1] Add close stations to the data model

First Pull request in the #86 PR-Train
@quarz12 I have decided that this part of the code could be better maintained upstream
=> Rebasing of https://github.com/TUM-Dev/NavigaTUM/pull/663 will be nessesary

## Proposed Changes (include Screenshots if possible)

- Added a scraper to collect mvg stations
- add them to each room/building

## How to test this PR

1. Select a room
2. See if the stations are asigned correctly

## How has this been tested?

- see above

## Checklist
